utils/wordList.js

```diff
@@ -0,0 +1 @@
sugar","witness","greed","shake","neighborhood","Doritos","doctor","Minotaur","Reddit","poppy","nail polish","Pikachu","Excalibur","cow","throne","braces","retail","badger","fizz","Elmo","yawn","Mars","quilt","firefighter","oil","bus","Anubis","floodlight","mayor","ravioli","cap","match","Asterix","arch","Goofy","tampon","building","pin","Tower of Pisa","Jenga","vacation","sauna","Tetris","nest","bunk bed","ABBA","scary","arm","iPhone","Susan Wojcicki","nun","parade","crawl space","Pac-Man","soil","lens","gas","mall","Green Lantern","celebrity","promotion","hammock","stereo","pillar","Capricorn","facade","Titanic","dinner","Luigi","Winnie the Pooh","Gandalf","Charlie Chaplin","villain","Johnny Bravo","halo","pigeon","butt cheeks","hip hop","macho","hovercraft","needle","hair roller","Notch","upgrade","bean bag","bellow","plunger","Mario","pigsty","KFC","wheel","Brazil","swag","Nasa","autograph","hilarious","acne","backpack","manhole","applause","Bruce Lee","Segway","advertisement","Poseidon","reception","vinegar","geography","peasant","Skittles","teacher","bed sheet","abyss","corner","cat","Miniclip","saxophone","safe","model","trapdoor","copper","anvil","Leonardo da Vinci","lounge","plumber","eel","blanket","house","otter","Mr. Meeseeks","kazoo","think","youtuber","frostbite","octagon","Bible","figurine","pear","world","peanut","daughter","Pringles","gravity","willow","sew","Northern Lights","health","backflip","Mont Blanc","crucible","Bugs Bunny","cement","actor","comedy","chainsaw","virtual reality","England","polar bear","fern","folder","microscope","tattoo","lynx","shopping","violin","Mr. Bean","Velociraptor","macaroni","quicksand","UFO","patient","glue","wrestler","duel","Sudoku","cushion","walnut","drama","caviar","King Kong","USB","Darwin Watterson","saddle","paintball","baklava","hermit","diploma","candle","Madagascar","beetle","Morty","dashboard","nutmeg","Hercules","Solar System","camera","tuba","leech","search","Slinky","lipstick","headache","Paris","pogo stick","lemur","Mr Meeseeks","player","invention","bumper","handle","sword","tabletop","shark","laundry","Chinatown","Spain","label","vanish","nose ring","drought","policeman","Stegosaurus","dice","pope","attic","Family Guy","server","vulture","banker","treadmill","chestnut","scarecrow","lake","doll","impact","papaya","landlord","flying pig","brunette","flask","notebook","boat","bookmark","floppy disk","back pain","shrew","T-rex","motherboard","stylus","magnifier","Band-Aid","journey","minivan","lotion","cartoon","skates","barcode","conversation","maid","desperate","Chrome","waterfall","furniture","Yin and Yang","wig","food","sensei","Venus","albatross","vegetarian","Florida","Milky Way","aristocrat","flush","tumor","safari","catalog","veterinarian","high score","afterlife","logo","mushroom","mole","marmalade","eraser","gummy bear","comet","hunter","bouncer","exercise","commander","classroom","lava","koala","market","miner","marble","fur","raincoat","sunburn","robber","leave","girl"]
```

Comment:
william wallace is a word here ???

Reply:
yea i grabbed another github's json list and transformed it into an array.
i found it tedious for our purposes to create a list of words from scratch